stubs/requests/requests/packages.pyi

```diff
@@ -1,3 +1,6 @@
 # requests also imports urllib3, idna, and chardet below
 # requests.packages. The stubs don't reflect that and it's recommended to
 # import these packages directly if needed.
+mod: str
+package: str
+target: str
```

Comment:
These are all loop variables that leak into the global scope. If stubtest complains about these with `ignore_missing_stub` set to `False`, the proper solution is to add entries to the allowlist with a comment explaining why they're being allowlisted. We definitely shouldn't be adding these to the stub.